python/cugraph/cugraph/datasets/__init__.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+# datasets module
+from cugraph.datasets.dataset import (
+    Dataset,
+    load_all,
+    set_download_dir,
+    get_download_dir,
+    default_download_dir,
+)
+from cugraph.datasets import metadata
+
+# metadata path for .yaml files
+meta_path = Path(__file__).parent / "metadata"
+
+# invidual datasets
+cyber = Dataset(meta_path / "cyber.yaml")
+dolphins = Dataset(meta_path / "dolphins.yaml")
+email_Eu_core = Dataset(meta_path / "email-Eu-core.yaml")
+karate = Dataset(meta_path / "karate.yaml")
+karate_asymmetric = Dataset(meta_path / "karate_asymmetric.yaml")
+karate_disjoint = Dataset(meta_path / "karate-disjoint.yaml")
+ktruss_polbooks = Dataset(meta_path / "ktruss_polbooks.yaml")
```

Comment:
I removed a few of the karate derivatives last time around. Afterwards, we talked about keeping these three around because I saw that they were frequently used in our unit tests. Should I still remove them?